python/templates/macros/implementations.jinja2

```diff
@@ -19,7 +19,24 @@
 }
 
 Mutable{{ type }} {{ full_type }}::clone() const {
+{% if prefix %}
   return Mutable{{ type }}(podio::utils::MaybeSharedPtr(new {{ type }}Obj(*m_obj), podio::utils::MarkOwned));
+{% else %}
+  auto tmp = new {{ type }}Obj();
+  tmp->id = podio::ObjectID();
+  tmp->data = m_obj->data;
```

Comment:
```suggestion
  auto tmp = new {{ type }}Obj(podio::ObjectID{}, m_obj->data);
```
This should be a public constructor that replaces **exactly these three lines**, i.e. it does not initialize any of the internal relations.

Reply:
Changed